index.html

```diff
@@ -684,7 +684,15 @@ <h2 class="heading-title">Sign up for our newsletter</h2>
         </div><!--//.ROW-->
       </div><!-- //.CONTAINER -->
       {% include modal-form.html style='dotted' initiative='newsletter' link-text='Sign up'%}
+      <div class="lgx-heading lgx-heading-white">
+        <iframe
+            src="https://anchor.fm/awesomeinc"
+            style="height:400px;width:800px;display:inline-block;align-items:center;justify-content:center;"
```

Comment:
@imabigboynow Can you change this so that it isn't using inline-styling? This will allow to make it responsive. Steps include:
1. Give the div a `class`
2. Create the styling (probably best to be in the `/sass/components/_subscribe.scss` file) 
3. Create responsive styling using the mixins at the top of the `/sass/helpers/_elements.scss` file.

We can chat more about this if this isn't clear.